src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -426,16 +426,38 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environment and system noise. Their
-collection is RECOMMENDED, before/during/after each session. This data is stored
-inside a subject folder named `sub-emptyroom`. The `session label` SHOULD be
-that of the date of the empty-room recording (e.g. `ses-YYYYMMDD`). The
-`scans.tsv` file containing the date/time of the acquisition SHOULD also be
-included. Hence, users will be able to retrieve the empty-room recording that
-best matches a particular session with a participant, based on date/time of
-recording.
+Empty-room MEG recordings capture the environmental and recording system's
+noise.
+There are two widely used strategies for collecting these recordings:
 
-Example:
+1.  Some labs record one empty-room measurement per day, which is then shared by
+    all researchers that record subject data on that day
+
+1.  Some labs make it the responsibility of each researcher to record a short
```

Comment:
```suggestion
2.  Some labs make it the responsibility of each researcher to record a short
```

Reply:
@hoechenberger this is a markdown "feature" for enumerated lists.

See this:
```
1. hey
1. ho
1. lets
1. go
```

becomes this:
1. hey
1. ho
1. lets
1. go

Reply:
Ok, sorry, didn't know that! Quite neat, makes inserting new elements easier!